lib/rucio/core/did.py

```diff
@@ -2734,6 +2734,7 @@ def set_status(scope, name, session=None, **kwargs):
     :param kwargs:  Keyword arguments of the form status_name=value.
     """
     statuses = ['open', ]
+    reevaluate_close_did = config_get_bool('subscriptions', 'reevaluate_dids_at_close', raise_exception=False, default=False, session=session)
```

Comment:
This is nitpicking, but wouldn’t it be better if the variable name matches the configuration?

Reply:
What do you mean ?

Reply:
I’m suggesting to rename `reevaluate_close_did` (the local variable) to `reevaluate_dids_at_close`.

Reply:
Sigh...

Reply:
I understand. Apologies for not catching this earlier.